index.html

```diff
@@ -1173,12 +1173,23 @@ <h3>Verification Material</h3>
             </p>
 
             <p>
+Implementers are often exposed to many different verification material formats
+such as
+<a href="https://en.wikipedia.org/wiki/Secure_Shell">SSH public keys</a>,
+<a href="https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail">
+PEM-encoded public keys</a>,
+hex-encoded public keys,
+<a data-cite="?RFC7517">JSON Web Keys</a>,
+<a href="#Multikey">Multikeys</a>, and
+<a data-cite="?8152#section-7">COSE public keys</a>.
 To increase the likelihood of interoperable implementations, this specification
-limits the number of formats for expressing verification material in a
-[=controller document=]. The fewer formats that implementers have to
-choose from, the more likely that interoperability will be achieved. This
-approach attempts to strike a delicate balance between easing implementation
-and providing support for formats that have historically had broad deployment.
+suggests limiting the number of formats used to express verification material in
+a [=controller document=]. The fewer formats that implementers have to choose
+from, the more likely that interoperability will be achieved for a particular
+ecosystem. Ideally, there would be one public key format, however, history has
```

Comment:
```suggestion
ecosystem. Ideally, there should be one public key format per application.
```

Reply:
We can't use "should" -- it's a reserved word. We also can't change it to what you're suggesting above as that was not where we had consensus at W3C TPAC. Checking the minutes linked to from the issue, we had consensus on:

* It's not realistic to support only one key format in the spec.
* We should urge developers to minimize the number of key formats they use.
* We should acknowledge the reality that implementers often have to use multiple key formats in a given application or ecosystem.

I can't take your change request because it goes against the discussion we had at W3C TPAC 2024. We'll discuss this on the next call.

Reply:
I'm not going to block either of these sentences, but what is "ideal" here is debatable since it's multidimensional.

It's certainly "ideal for interop implementers to write as little software as possible to efficiently interoperate". But, of course, some applications may more greatly benefit from having feature X use format 1 and feature Y use format 2 ... and we just can't know that at this layer. Maybe some feature in an application is impractical or infeasible if you use format 1 and the inverse is true for another feature in the same application with format 2 ... but allowing two formats solves this. In other words, it's good that this layer doesn't make some applications or features too infeasible/impractical whilst encouraging minimizing formats for interoperability purposes.

Reply:
Well, strictly speaking, "_SHOULD_" is for normative language, "should" is not. 

But I agree that we should (sic!) not reopen the discussion.

Reply:
Incorporated the above thinking into [my suggestion, below](https://github.com/w3c/controller-document/pull/120/files#r1846869576).